__tests__/normalize-manifest.js

```diff
@@ -15,60 +15,71 @@ jasmine.DEFAULT_TIMEOUT_INTERVAL = 60000;
 
-for (const name of nativeFs.readdirSync(fixturesLoc)) {
-  if (name[0] === '.') {
-    continue;
-  }
+async function _compareManifests(loc, options = {}): Promise<void> {
+  const actualWarnings = [];
+  const expectedWarnings = options.expectedWarnings || (await fs.readJson(path.join(loc, 'warnings.json')));
+  const reporter = new NoopReporter();
 
-  const loc = path.join(fixturesLoc, name);
-
-  test(name, async () => {
-    const actualWarnings = [];
-    const expectedWarnings = await fs.readJson(path.join(loc, 'warnings.json'));
+  // $FlowFixMe: Investigate
+  reporter.warn = function(msg) {
+    actualWarnings.push(msg);
+  };
 
-    const reporter = new NoopReporter();
+  const config = await Config.create({cwd: loc, globalFolder: options.globalFolder}, reporter);
 
-    // $FlowFixMe: Investigate
-    reporter.warn = function(msg) {
-      actualWarnings.push(msg);
-    };
+  let actual = await fs.readJson(path.join(loc, 'actual.json'));
+  const expected = await fs.readJson(path.join(loc, 'expected.json'));
 
-    const config = await Config.create({cwd: loc}, reporter);
+  let isRoot = actual._root;
+  if (isRoot == null) {
+    isRoot = true;
+  } else {
+    delete actual._root;
+  }
 
-    let actual = await fs.readJson(path.join(loc, 'actual.json'));
-    const expected = await fs.readJson(path.join(loc, 'expected.json'));
+  const error = expected._error;
+  if (error) {
+    delete expected._error;
+  }
 
-    let isRoot = actual._root;
-    if (isRoot == null) {
-      isRoot = true;
+  try {
+    actual = await normalizeManifest(actual, loc, config, isRoot);
+  } catch (err) {
+    if (error && err.message === error) {
+      return;
     } else {
-      delete actual._root;
+      throw err;
     }
+  }
 
-    const error = expected._error;
-    if (error) {
-      delete expected._error;
-    }
+  if (error) {
+    throw new Error(`Expected to throw error: ${error}`);
+  }
 
-    try {
-      actual = await normalizeManifest(actual, loc, config, isRoot);
-    } catch (err) {
-      if (error && err.message === error) {
-        return;
-      } else {
-        throw err;
-      }
-    }
+  expect(map(actual)).toEqual(expand(expected));
+  expect(actualWarnings).toEqual(expectedWarnings);
+}
 
-    if (error) {
-      throw new Error(`Expected to throw error: ${error}`);
-    }
+for (const name of nativeFs.readdirSync(fixturesLoc)) {
+  if (name[0] === '.') {
+    continue;
+  }
 
-    expect(map(actual)).toEqual(expand(expected));
-    expect(actualWarnings).toEqual(expectedWarnings);
-  });
+  const loc = path.join(fixturesLoc, name);
+
+  test(name, _compareManifests.bind(null, loc));
 }
 
+const globalfixturesLoc = path.join(__dirname, 'fixtures', 'normalize-manifest', 'name');
+
+test(
+  'license warnings should not be thrown with global commands',
+  _compareManifests.bind(null, globalfixturesLoc, {
+    expectedWarnings: [],
+    globalFolder: globalfixturesLoc,
+  }),
+);
+
```

Comment:
Ok! :) I was trying to understand what changed in this file - I can see now that you've factored the test so that we can call it multiple times with different parameters, perfect 👍